sros/docker/launch.py

```diff
@@ -322,24 +323,27 @@ def _parse(cfg, obj, skip_nics=False):
 
         for hw_part in cfg.split("___"):
             if "cp: " in hw_part:
-                variant["cp"] = _parse(hw_part.strip(), None, skip_nics=True)
+                variant["cp"] = parse_variant_line(
+                    hw_part.strip(), None, skip_nics=True
+                )
             elif "lc: " in hw_part:
-                lc = _parse(hw_part.strip(), None)
+                lc = parse_variant_line(hw_part.strip(), None)
                 variant["lcs"].append(lc)
 
         # Sort lc line by slot number
         variant["lcs"] = sort_lc_lines_by_slot(variant["lcs"])
     else:
         # parsing integrated mode config
         variant["deployment_model"] = "integrated"
-        variant = _parse(cfg, obj=variant)
+        variant = parse_variant_line(cfg, obj=variant)
 
     return variant
 
 
 # sort line card definitions if they were provided in a non-consequetive order
 def sort_lc_lines_by_slot(lc_lines: list) -> list:
-    timos_tuples = [(v["slot"], v) for v in lc_lines]
+    max_slot = 99
```

Comment:
This part to ensure if no slot information then put it to the last sequence in tuples